server/src/main/java/io/crate/replication/logical/action/UpdateSubscriptionAction.java

```diff
@@ -169,7 +167,7 @@ protected ClusterBlockException checkBlock(Request request,
     }
 
 
-    public static class Request extends MasterNodeReadRequest<Request> {
+    public static class Request extends MasterNodeReadRequest<Request> implements AckedRequest {
```

Comment:
This looks initially wrong as this action is not only reading but writing. Correct fix imho is changing to extend from `AcknowledgedRequest`.
```suggestion
    public static class Request extends AcknowledgedRequest<Request> {
```